src/components/TransactionDialog/TransactionDialogContent.tsx

```diff
@@ -2,13 +2,83 @@ import { FC, Fragment, useEffect, useRef, useState } from "react";
 import { useChain, useManager } from "@cosmos-kit/react";
 import { ArrowLeftIcon, CheckCircleIcon } from "@heroicons/react/20/solid";
 import Toast from "@/elements/Toast";
-import { executeRoute } from "@/solve/form";
 import RouteDisplay from "../RouteDisplay";
-import { Route } from ".";
-import { useSkipClient } from "@/solve";
+import { RouteResponse, useSkipClient } from "@/solve";
+import { useToast } from "@/context/toast";
+import { getChainByID } from "@/utils/utils";
+import { executeRoute } from "@/solve/execute-route";
+import { useAssets } from "@/context/assets";
+import { Chain, useChains } from "@/context/chains";
+
+const TransactionSuccessView: FC<{
+  route: RouteResponse;
+  onClose: () => void;
+}> = ({ route, onClose }) => {
+  const { getAsset } = useAssets();
+  const { chains } = useChains();
+
+  const sourceAsset = getAsset(
+    route.source_asset_denom,
+    route.source_asset_chain_id
+  );
+  const destinationAsset = getAsset(
+    route.dest_asset_denom,
+    route.dest_asset_chain_id
+  );
+
+  const sourceChain = chains.find(
+    (c) => c.chain_id === route.source_asset_chain_id
+  ) as Chain;
+  const destinationChain = chains.find(
+    (c) => c.chain_id === route.dest_asset_chain_id
+  ) as Chain;
+
+  return (
+    <div className="flex flex-col items-center h-full px-4 py-6 pt-28 overflow-y-auto scrollbar-hide">
+      <div className="text-emerald-400">
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 24 24"
+          fill="currentColor"
+          className="w-[100px] h-[100px]"
+        >
+          <path
+            fillRule="evenodd"
+            d="M8.603 3.799A4.49 4.49 0 0112 2.25c1.357 0 2.573.6 3.397 1.549a4.49 4.49 0 013.498 1.307 4.491 4.491 0 011.307 3.497A4.49 4.49 0 0121.75 12a4.49 4.49 0 01-1.549 3.397 4.491 4.491 0 01-1.307 3.497 4.491 4.491 0 01-3.497 1.307A4.49 4.49 0 0112 21.75a4.49 4.49 0 01-3.397-1.549 4.49 4.49 0 01-3.498-1.306 4.491 4.491 0 01-1.307-3.498A4.49 4.49 0 012.25 12c0-1.357.6-2.573 1.549-3.397a4.49 4.49 0 011.307-3.497 4.49 4.49 0 013.497-1.307zm7.007 6.387a.75.75 0 10-1.22-.872l-3.236 4.53L9.53 12.22a.75.75 0 00-1.06 1.06l2.25 2.25a.75.75 0 001.14-.094l3.75-5.25z"
+            clipRule="evenodd"
+          />
+        </svg>
+      </div>
+      <div>
+        <p className="font-bold text-3xl mb-4">
+          {route.does_swap ? "Swap" : "Transfer"} Successful
+        </p>
+      </div>
+      <p className="font-medium text-neutral-400 pb-8 flex-1 text-center">
+        {route.does_swap &&
+          `Successfully swapped ${
+            sourceAsset?.symbol ?? route.source_asset_denom
+          } for ${destinationAsset?.symbol ?? route.dest_asset_denom}`}
+        {!route.does_swap &&
+          `Successfully transfered ${
+            sourceAsset?.symbol ?? route.source_asset_denom
+          } from ${sourceChain.prettyName} to ${destinationChain.prettyName}`}
```

Comment:
Where are you getting pretty name from? LMK if this is coming from somewhere outside of the API

Reply:
Looks like it comes from chain-registry, our chains endpoint has a chain-name but it's not capitalized.

I assume he is using it this way because the chain registry one is easy to access.